index.js

```diff
@@ -0,0 +1,51 @@
+const GIPHY_URL = 'https://api.giphy.com/v1/stickers/search?'; 
+const API_KEY = 'api_key=E26iZ0BIEszi7RA4GeEmXzJPRpkSeQ6m';
+const SEARCH_PARAM = '&q=';
+const AMOUNT = '&limit=50';
+
+let numberElements = 5;
+let userWord;
+let url;
+let json;
+
+const input = document.getElementById('findInput');
+const imgWrap = document.querySelector('.wrap-for-copy').cloneNode(true);
+
+document.querySelector('.wrap-for-copy').remove();
+imgWrap.classList.remove('wrap-for-copy');
+
+const replaceImage = () => {
+  //shuffle array-------------
+  let array = [...json.data];
+  array.sort(function() {
+      return Math.random() - 0.5
+  });
+  //--------------------------
+  const needGifArr = array.slice(0, numberElements)
+  needGifArr.forEach(gif => {
+    const newImg = imgWrap.cloneNode(true);
+    newImg.querySelector('img').src = gif.images.original.url;
+    newImg.querySelector('img').alt = gif.title;
+    document.getElementById('listImages').appendChild(newImg);
+  })
+}
+
+const start = async function() {
```

Comment:
ES6 function declaration can be used

Reply:
it should accept userWord as a parameter and build URL inside

Reply:
this function mixes responsibility. It makes request to the server and calls rerender. Two options:
1) this function just make request and returns data. In other place we wait for response and rerender using a new data
2) this function calls another function that provides data from server and calls function to make rendering using this data